extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -940,6 +940,10 @@ export class Carousel {
    * @private
    */
   isScrollAtEndingEdge_() {
+    // Lightbox and trivial case
+    if (!this.mixedLength_ && this.visibleCount_ === 1) {
+      return this.getCurrentIndex() >= this.slides_.length - 1;
+    }
```

Comment:
Shouldn't this check go in `isAtEnd` instead of `isScrollAtEndingEdge_`, since that is where the `loop` guard is also in place?

Reply:
Great idea!